common/include/pcl/type_traits.h

```diff
@@ -283,12 +283,34 @@ namespace pcl
 #endif
 
   /**
-   * \todo: Remove in C++17
+   * \todo: Remove in C++20
    */
 #ifndef __cpp_lib_remove_cvref
   template <typename T>
   using remove_cvref_t = std::remove_cv_t<std::remove_reference_t<T>>;
 #else
   using std::remove_cvref_t;
 #endif
+
+  /**
+  * \todo: Remove in C++17
+  */
+#ifndef __cpp_lib_void_t
+  template <typename...>
+  using void_t = void;
+#else
+  using std::void_t;
+#endif
+
+  /**
+   * \todo: Remove in C++17
+   */
+  // Implementation taken from
+  // https://stackoverflow.com/questions/31533469/check-a-parameter-pack-for-all-of-type-t
+  template <typename... Conds>
```

Comment:
Brief documentation please

Reply:
Is it needed in this case? Since its already part of C++17 and not something PCL specific.